lib/index.js

```diff
@@ -30,6 +42,9 @@ function createApp (opts) {
 
   app.get('/__bundle/:file.bundle.js', bundleFileRoute)
 
+  // Inject livereload snippet
+  app.use(require('connect-livereload')())
```

Comment:
This line injects the livereload snippet in any arbitrary html file, so if folks write their own `foo.html` then this will drop in the livereload capability.